app/styles/app/modules/sync-button.sass

```diff
@@ -18,15 +18,15 @@
     right: 9em
     top: 0
     width: 170%
-    z-index: -1
+    @include z-index(under)
     transition: transform 150ms ease, opacity 250ms ease
     transform: translateX(80%)
     opacity: 0
     background: white
   .button:hover ~ .sync-last
     transform: translateX(0)
     opacity: 1
-    z-index: 1
```

Comment:
![sync](https://user-images.githubusercontent.com/43280/31220476-da666178-a9c0-11e7-8998-76e35e244693.gif)

I changed the layering and it’s back.